conversation/data-store-status-checker/data_store_checker.ipynb

```diff
@@ -1,729 +1,797 @@
-      ]
-    },
-    {
-      "cell_type": "markdown",
-      "metadata": {
-        "id": "c56c36b2df57"
-      },
-      "source": [
-        "# Check Data Store Index Status\n",
-        "\n",
-        "Using the `list_documents` method, we can do a check to see if the data store has finished indexing.\n"
-      ]
-    },
-    {
-      "cell_type": "code",
-      "execution_count": 9,
-      "metadata": {
-        "id": "86faaf544106"
-      },
-      "outputs": [
-        {
-          "name": "stdout",
-          "output_type": "stream",
-          "text": [
-            "\n",
-            "No docs found.\n",
-            "\n",
-            "It's likely one of two issues: \n",
-            "  [1] Your data store is not finished indexing. \n",
-            "  [2] Your data store failed indexing.\n",
-            "\n",
-            "If you just added your data store, it can take up to 4 hours before it will become available.\n",
-            "\n"
-          ]
-        }
-      ],
-      "source": [
-        "docs = list_documents(project_id, location, datastore_id)\n",
-        "\n",
-        "if len(docs) == 0:\n",
-        "    print(PENDING_MESSAGE)\n",
-        "else:\n",
-        "    SUCCESS_MESSAGE = f\"\"\"\n",
-        "  Success! 🎉\\n\n",
-        "  Your indexing is complete.\\n\n",
-        "  Your index contains {len(docs)} documents.\n",
-        "  \"\"\"\n",
-        "    print(SUCCESS_MESSAGE)"
-      ]
-    },
-    {
-      "cell_type": "markdown",
-      "metadata": {
-        "id": "f957771b340a"
-      },
-      "source": [
-        "# List Documents\n",
-        "\n",
-        "List all the documents for a given Data Store ID\n"
-      ]
-    },
-    {
-      "cell_type": "code",
-      "execution_count": 6,
-      "metadata": {
-        "id": "1fa7c648a7ae"
-      },
-      "outputs": [
-        {
-          "data": {
-            "text/plain": [
-              "struct_data {\n",
-              "}\n",
-              "name: \"projects/772105163160/locations/global/collections/default_collection/dataStores/cgc_1690226759325/branches/0/documents/000a98558b6fe9aef7992c9023fb7fdb\"\n",
-              "id: \"000a98558b6fe9aef7992c9023fb7fdb\"\n",
-              "schema_id: \"default_schema\"\n",
-              "content {\n",
-              "  uri: \"https://cloud.google.com/docs/security/data-loss-prevention/revoking-user-access?hl=es-419\"\n",
-              "  mime_type: \"text/html\"\n",
-              "}\n",
-              "parent_document_id: \"000a98558b6fe9aef7992c9023fb7fdb\""
-            ]
-          },
-          "execution_count": 6,
-          "metadata": {},
-          "output_type": "execute_result"
-        }
-      ],
-      "source": [
-        "docs = list_documents(project_id, location, datastore_id)\n",
-        "docs[0]"
-      ]
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "id": "7be72547515d"
+   },
+   "source": [
+    "# Vertex AI Conversation Data Store Status Checker\n",
```

Comment:
Should this be `Vertex AI Search and Conversation Data Store Status Checker` to make it clear in the title that it can be used for both?